src/comparator/message_comparator.py

```diff
@@ -84,7 +82,8 @@ def _compare(self, message_original, message_update):
             )
 
         # 6. Check `google.api.resource` annotation.
-        self._compare_resources(message_original.resource, message_update.resource)
+        # This check has been done in file_set comparator. Since we have
+        # registered all resources in the database. 
```

Comment:
Remove the resource checking in message comparator, because the message-level resources have been registered at file_set database and compared in the file_set comparator.